docs/pages/introduction/why-not-expo.mdx

```diff
@@ -4,107 +4,40 @@ sidebar_title: Limitations
-
-Apps built with `expo build:ios|android` [contain code for the entire Expo SDK](https://expo.fyi/managed-app-size), you cannot customize the native dependencies, including Facebook's Audience Network library, so if you build your app this way you cannot designated it as "designed primarily for children under 13" in the App Store or Play Store, _even though this code is never run unless you explicitly call it_.
-
-Apps built with `eas build -p ios|android` [include only your app's explicit native dependencies](https://blog.expo.dev/expo-managed-workflow-in-2021-d1c9b68aa10), and so this limitation does not apply if you use EAS Build. [Learn about how to use it](/build/introduction).
-
-</Collapsible>
-
-> We are either actively working on or planning to build solutions to all of the limitations listed above, and if you think anything is missing, please bring it to our attention by posting to our [feature requests board](https://expo.canny.io/feature-requests) or the [forums](https://forums.expo.dev/).
-
-## Limitations of the bare workflow
-
-In the bare workflow, we have full access to the underlying native projects and any native code. It's a "bare" native project with React Native and one or more packages from the Expo SDK installed. Anything that you can do in a native project is possible here.
-
-The following list is therefore specifically oriented towards the limitations that exist around using Expo tools and services in the bare workflow.
-
-<Collapsible summary={<span><span className="strike">Build service only works in the managed workflow</span> (✅ Resolved in December 2020)</span>}>
-
-~~To build your app binaries for distribution on the Apple App Store and Google Play Store you will need to follow the same steps that you would in any native project, the Expo build service can't handle it for you. We are working on bringing bare workflow support to the build service in the near future.~~
-
-You can now use [EAS Build](/build/introduction) to build and sign your apps just as easily as in the managed workflow! [Read the announcement blog post](https://blog.expo.dev/expo-application-services-eas-build-and-submit-fc1d1476aa2e).
-
-</Collapsible>
-
-<Collapsible summary="Configuration must be done on each native project rather than once with app.json">
-
-Configuring app icons, launch screen, and so on must be configured in the native projects for each platform using the standard native tooling, rather than once using a simple JSON object.
-
-</Collapsible>
+**All of the above have been addressed with EAS Build which is compatible with all type of React Native projects. If you come across them, you can transition your project to use [development builds](/development/introduction/).**
```

Comment:
```suggestion
**We strongly recommend any projects that may need additional libraries with native code migrate to [development builds](/development/introduction/). It's like a version of Expo Go specifically customized to your app's needs.**
```
Or insert verbiage from your new dev client doc :-). At any rate, I think this should reflect that dev builds are for everyone.